pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "visier-connector-python"
+authors = [
+    {name = "Visier Research & Development", email = "info@visier.com"},
+]
+description = """Visier People Data connector"""
```

Comment:
There's no long description in the new file. Intentional?

Reply:
Yes, the long description is replaced by the reference to the `README.md`